### PR TITLE
fix 8.0 branch travis build

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by gurneyalex** [comment]
Build should be green when https://github.com/OCA/maintainer-quality-tools/pull/309 is merged.

**Comment by lmignon** [comment]
:+1: Once travis is green

File: .travis.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ install:
   - pip install python-ldap
   - pip install unidecode
   - pip install validate_email
-  - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
-  - ln -s server_environment_files_sample ./server_environment_files
+  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
+  - ln -s ~/build/OCA/server-tools/server_environment_files_sample ~/build/OCA/server-tools/server_environment_files
```
Comment (pedrobaeza) [inline .travis.yml]: As said by @moylop260, this should be `TRAVIS_BUILD_DIR` (with the proper shell notation).

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**